src/query/api/v1/httpd/handler.go

```diff
@@ -22,11 +22,16 @@ package httpd
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
 	"net/http"
 	_ "net/http/pprof" // needed for pprof handler registration
+	"os"
+	"strings"
 	"time"
 
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/zap"
```

Comment:
nit: github.com/m3db/m3 imports should be grouped as second group, then all third party exports as a separate import group below that. The linter is supposed to catch this but I think we need to look at the linter since it seems to not be running on CI lately...